bundle/compliance/cis_gcp/rules/cis_4_5/data.yaml

```diff
@@ -0,0 +1,95 @@
+metadata:
+  id: d63a2fd8-7ba2-5589-9899-23f99fd8c846
+  name: Ensure ‘Enable Connecting to Serial Ports’ Is Not Enabled for VM Instance
+  profile_applicability: '* Level 1'
+  description: |-
+    Interacting with a serial port is often referred to as the serial console, which is similar to using a terminal window, in that input and output is entirely in text mode and there is no graphical interface or mouse support.
+
+    If you enable the interactive serial console on an instance, clients can attempt to connect to that instance from any IP address.
+    Therefore interactive serial console support should be disabled.
+  rationale: |-
+    A virtual machine instance has four virtual serial ports.
+    Interacting with a serial port is similar to using a terminal window, in that input and output is entirely in text mode and there is no graphical interface or mouse support.
+    The instance's operating system, BIOS, and other system-level entities often write output to the serial ports, and can accept input such as commands or answers to prompts.
+    Typically, these system-level entities use the first serial port (port 1) and serial port 1 is often referred to as the serial console.
+
+    The interactive serial console does not support IP-based access restrictions such as IP whitelists.
+    If you enable the interactive serial console on an instance, clients can attempt to connect to that instance from any IP address.
+    This allows anybody to connect to that instance if they know the correct SSH key, username, project ID, zone, and instance name.
+
+    Therefore interactive serial console support should be disabled.
+  audit: |-
+    **From Google Cloud CLI**
+
+    1. Login to Google Cloud console
+    2. Go to Computer Engine
+    3. Go to VM instances
+    4. Click on the Specific VM
+    5. Ensure `Enable connecting to serial ports` below `Remote access` block is unselected.
+
+    **From Google Cloud Console**
+
+    Ensure the below command's output shows `null`:
+
+    ```
+    gcloud compute instances describe <vmName> --zone=<region> --format="json(metadata.items[].key,metadata.items[].value)"
+    ``` 
+
+    or `key` and `value` properties from below command's json response are equal to `serial-port-enable` and `0` or `false` respectively.
+
+    ```
+     {
+     "metadata": {
+     "items": [
+     {
+     "key": "serial-port-enable",
+     "value": "0"
+     }
+     ]
+     }
+     }
+    ```
```

Comment:
we need to format the json values somehow, they will look weird in Kibana

Reply:
added a new function to address such issues.